### PR TITLE
Add a "What weird day is it" announcement 

### LINKED ISSUE #275: Feature: Add a daily "what weird day is it" announcement
At midnight or so the bot would say what celebratory day it is.
e.g for today it would say something like
`Today is National Lima Bean Respect Day`

### PR DESCRIPTION
**Pull request by ldgrp**
Posts a message on general with a holiday from https://www.timeanddate.com/holidays/fun/ at 9am.

Unfortunately, I couldn't think of a better name for the script. 

Fixes #275.

### CODE REVIEW

**Comment by gricey432** [comment]
Lgtm.
Some good but non-critical improvements would be some tests, and replacing your magic dictionary with a `namedtuple` or a class

File: uqcsbot/scripts/holiday.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+@bot.on_schedule('cron', hour=14, minute=45, timezone='Australia/Brisbane')
+def holiday() -> None:
+    '''
+    Posts a random celebratory day on #general from
+    https://www.timeanddate.com/holidays/fun/
+    '''
+    channel = bot.channels.get("general")
+
+    now = datetime.now().strftime("%d %b").lstrip('0')
```
Comment (mitchmcdee) [inline uqcsbot/scripts/holiday.py]: is this used anywhere anymore?

File: uqcsbot/scripts/holiday.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    def is_today(self) -> bool:
+        '''
+        Returns true if the holiday is celebrated today
+        '''
+        now = datetime.now()
+        return self.date.month == now.month and \
+                self.date.day == now.day
+
+@bot.on_schedule('cron', hour=14, minute=45, timezone='Australia/Brisbane')
```
Comment (mitchmcdee) [inline uqcsbot/scripts/holiday.py]: Is this a time left over from testing? I think you had it at 9 earlier? :^)

File: uqcsbot/scripts/holiday.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        self.date = date
+        self.description = description
+        self.url = url
+
+    def is_today(self) -> bool:
+        '''
+        Returns true if the holiday is celebrated today
+        '''
+        now = datetime.now()
+        return self.date.month == now.month and \
```
Comment (mitchmcdee) [inline uqcsbot/scripts/holiday.py]: this can probably fit on one line and be < 100 chars no worries

File: uqcsbot/scripts/holiday.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+    
+def get_holidays_from_page(holiday_page) -> list:
+    '''
+    Strips results from html page
+    '''
+    soup = BeautifulSoup(holiday_page, 'html.parser')
+    soup_holidays = soup.find_all(class_="c0") + \
+                    soup.find_all(class_="c1") + \
+                    soup.find_all(class_="hl")
```
Comment (mitchmcdee) [inline uqcsbot/scripts/holiday.py]: what is the `hl` class?

File: uqcsbot/scripts/holiday.py (hunk 1 above)
Comment (mitchmcdee) [reply uqcsbot/scripts/holiday.py]: if not, it + other references can be removed :)

**Pull request merged**